docs/tutorials/exoscale.md

```diff
@@ -0,0 +1,155 @@
+# Setting up ExternalDNS for Exoscale
+
+## Prerequisites
+
+Exoscale provider support was added via [this PR](https://github.com/kubernetes-incubator/external-dns/pull/625), thus you need to use external-dns v0.5.5.
+
+The Exoscale provider expects that your Exoscale zones, you wish to add records to, already exists
+and are configured correctly. It does not add, remove or configure new zones in anyway.
+
+To do this pease refer to the [Exoscale DNS documentation](https://community.exoscale.com/documentation/dns/).
+
+Additionally you will have to provide the Exoscale...:
+
+* API Key
+* API Secret
+* API Endpoint
+* Elastic IP address, to access the workers
+
+## Deployment
+
+Deploying external DNS for Exoscale is actually nearly identical to deploying
+it for other providers. This is what a sample `deployment.yaml` looks like:
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: external-dns
+spec:
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: external-dns
+    spec:
+      # Only use if you're also using RBAC
+      # serviceAccountName: external-dns
+      containers:
+      - name: external-dns
+        image: registry.opensource.zalan.do/teapot/external-dns:v0.5.5
+        args:
+        - --source=ingress # or service or both
+        - --provider=exoscale
+        - --domain-filter={{ my-domain }}
+        - --policy=sync # if you want DNS entries to get deleted as well
+        - --txt-owner-id={{ owner-id-for-this-external-dns }}
+        - --exoscale-endpoint={{ endpoint }} # usually https://api.exoscale.ch/dns
+        - --exoscale-apikey={{ api-key}}
+        - --exoscale-apisecret={{ api-secret }}
+```
+
+## RBAC
+
+If your cluster is RBAC enabled, you also need to setup the following, before you can run external-dns:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: external-dns
+  namespace: default
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: external-dns
+rules:
+- apiGroups: [""]
+  resources: ["services"]
+  verbs: ["get","watch","list"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get","watch","list"]
+- apiGroups: ["extensions"]
+  resources: ["ingresses"]
+  verbs: ["get","watch","list"]
```

Comment:
Sorry I wasn't seeing it but you have to add 

```yaml
- apiGroups: [""]
  resources: ["nodes"]
  verbs: ["list"]
```

to run external-dns with RBAC